keel-web/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -67,12 +68,17 @@ class DefaultConfiguration(
   @Bean
   fun idGenerator(): ULID = ULID()
 
+  @Bean
+  fun jsonComponentModule() = JsonComponentModule()
+
   @Bean(name = ["jsonMapper", "objectMapper"])
   @Primary
-  fun objectMapper(): ObjectMapper = configuredObjectMapper()
+  fun objectMapper(jsonComponentModule: JsonComponentModule): ObjectMapper =
+    configuredObjectMapper().registerModule(jsonComponentModule)
```

Comment:
Example of wiring an `ObjectMapper` with `JsonComponentModule`, which itself needs to be a `@Bean` so it can find and register (de-)serializer beans.